### PR TITLE
Issue 2 update task redundancy

### PR DESCRIPTION
**Pull request by teleyinex**

### CODE REVIEW

File: helpers.py [hunk 1]
```diff
@@ -167,24 +212,29 @@ def find_app_by_short_name(short_name, pbclient):
         return response[0]
     except exceptions.ConnectionError:
         raise
-    except:
-        format_error("pbclient.find_app", response)
+    except ProjectNotFound:
+        raise
```
Comment (alejandrodob) [inline helpers.py]: what happens here with exceptions that are neither ConnectionError nor ProjectNotFound?

Comment (teleyinex) [inline helpers.py]: Good point. Fixing it.

File: test/test_pbs_add_tasks.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 from mock import patch, MagicMock
 from nose.tools import assert_raises
 from requests import exceptions
+from pbsexceptions import *
 
 class TestHelpers(TestDefault):
```
Comment (alejandrodob) [inline test/test_pbs_add_tasks.py]: Maybe change the name of this test class here

File: test/test_pbs_create_project.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 from mock import patch, MagicMock
 from nose.tools import assert_raises
 from requests import exceptions
+from pbsexceptions import ProjectNotFound
 
 
 class TestHelpers(TestDefault):
```
Comment (alejandrodob) [inline test/test_pbs_create_project.py]: change class name

File: test/test_pbs_delete_tasks.py [hunk 4]
```diff
@@ -6,6 +6,7 @@
 from mock import patch, MagicMock
 from nose.tools import assert_raises
 from requests import exceptions
+from pbsexceptions import *
 
 
 class TestHelpers(TestDefault):
```
Comment (alejandrodob) [inline test/test_pbs_delete_tasks.py]: change class name

File: test/test_pbs_update_project.py [hunk 5]
```diff
@@ -5,6 +5,7 @@
 from mock import patch, MagicMock
 from nose.tools import assert_raises
 from requests import exceptions
+from pbsexceptions import ProjectNotFound
 
 class TestHelpers(TestDefault):
```
Comment (alejandrodob) [inline test/test_pbs_update_project.py]: here too

File: test/test_pbs_update_task_redundancy.py [hunk 6]
```diff
@@ -0,0 +1,91 @@
+"""Test module for pbs client."""
+from helpers import _update_tasks_redundancy
+from default import TestDefault
+from mock import patch, MagicMock
+from nose.tools import assert_raises
+from pbsexceptions import *
+from requests import exceptions
+
+
+class TestHelpers(TestDefault):
```
Comment (alejandrodob) [inline test/test_pbs_update_task_redundancy.py]: and here :)

**Comment by alejandrodob** [comment]
ok, other than my comments about the test class names, :cake:

**Pull request merged**